package/crds/glue.aws.crossplane.io_jobs.yaml

```diff
@@ -61,6 +61,85 @@ spec:
               forProvider:
                 description: JobParameters defines the desired state of Job
                 properties:
+                  SecurityConfiguration:
```

Comment:
SecurityConfiguration vs. securityConfiguration ?

Reply:
Yes, good catch! I will fix this in the next commit!